java/meta/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+galaxy_info:
+  author: AeroGear
+  company: AeroGear
+  description: "Java for Red Hat family OS"
+  issue_tracker_url: https://issues.jboss.org/projects/AGDIGGER/issues
+  license: "license (Apache 2)"
+  min_ansible_version: "2.2.0"
+  platforms:
+    -
+      name: fedora
```

Comment:
Centos doesn't exist in the list of supported  platforms See https://galaxy.ansible.com/api/v1/platforms/. Lets leave as is